### ISSUE TITLE
Use @api3/eslint-plugin-commons for eslint

### ISSUE THREAD

**Issue by dcroote** [api3dao/airnode#2013]
A number of new rules require `parserServices` to be generated which results in the error:

```
You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser
```
Currently it isn't specified:

https://github.com/api3dao/airnode/blob/c43d981905ae27562e2488c6211dda4304f2c5ed/.eslintrc.js#L9-L12

Specifying this, however, isn't as easy as other repos given this monorepo structure and, I believe, how the packages' `tsconfig.json` files use `references`, potentially confusing ESLint on how to find the files.

Disabling these few rules allows it to run, but with quite a number of errors:

```sh
✖ 2921 problems (2921 errors, 0 warnings)
399 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by Siegrift** [comment]
Yeah, I think for Airnode we should disable many of the non-fixable rules just to make the migration simpler. These can always be re-enabled later as separate issues.
